pages/career/applyjob/[jobID].page.js

```diff
@@ -0,0 +1,474 @@
+import React, { useState, useEffect } from "react";
+import { useRouter } from "next/router";
+import { useJobContext } from "../../../context/JobContext";
+import Button from "../../../components/UI/Button/Button";
```

Comment:
`pages/career/applyjob/[jobID].page.js` ---> `pages/career/index.page.js`.  ( please can we have like this jobID=${jobId} we can query this param